database-plugins/src/main/java/io/cdap/plugin/db/batch/sink/DBSink.java

```diff
@@ -115,6 +117,9 @@ public void prepareRun(BatchSinkContext context) {
       DBUtils.cleanup(driverClass);
     }
     context.addOutput(Output.of(dbSinkConfig.referenceName, new DBOutputFormatProvider(dbSinkConfig, driverClass)));
+
+    recordLineage(context, dbSinkConfig.referenceName, Schema.of(Schema.Type.STRING),
+        Arrays.asList(dbSinkConfig.columns.split(",")));
```

Comment:
The field must be derived from the schema

Reply:
Also the indentation is incorrect. This line should align with the context like: 
 ```
recordLineage(context, dbSinkConfig.referenceName, Schema.of(Schema.Type.STRING), 
               outputFields)
```